mava/environment_loop.py

```diff
@@ -278,15 +279,15 @@ def step_executor() -> None:
 
             except Exception as e:
                 if self._executor._evaluator:
-                    print(
+                    tf.print(
```

Comment:
Can we not use `jax.debug.print`?

Reply:
Yip I agree. I initially tried using `jax.debug.print`, but it did not seem to work. So I am not sure if there is another way to print that does not use TensorFlow. Can you test if `tf.print` shows on your side when introducing a bug? You use a virtual environment right?